pushy/src/main/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -833,6 +933,10 @@ protected Map<String, Object> buildPayloadMap() {
                 if (this.launchImageFileName != null) {
                     alert.put(LAUNCH_IMAGE_KEY, this.launchImageFileName);
                 }
+
+                if (this.soundFileName != null && isLiveActivityPayload()) {
+                    alert.put(SOUND_KEY, this.soundFileName);
+                }
```

Comment:
Minor: I'd say this is worth a comment explaining the strange payload situation we discussed earlier. Without a  comment, my fear is that somebody (i.e. me in six months) will look at this and think "well that's strange," then try to "fix" it.